code/__HELPERS/global_lists.dm

```diff
@@ -1,8 +1,8 @@
-GLOBAL_LIST_EMPTY(metatips)
-GLOBAL_LIST_EMPTY(marinetips)
-GLOBAL_LIST_EMPTY(xenotips)
-GLOBAL_LIST_EMPTY(joketips)
-#define ALLTIPS (GLOB.marinetips + GLOB.xenotips + GLOB.joketips + GLOB.metatips)
+GLOBAL_LIST_INIT(metatips, file2list("strings/tips/marine.txt"))
```

Comment:
was done this way to avoid world load lag, not sure why its being changed back

Reply:
All the other strings get done this way, just like datum ref lists.

Reply:
datum ref lists is called during world/new this is before world/new

Reply:
ideally these should be loaded by a string subsystem init

Reply:
Check explanation for the change request.

Reply:
It should be done via init ,this is such a tiny little margin that it is out of scope to request an entirely new subsystem. Currently this fixes more issues than it presents.

Reply:
this isnt a fix pr its a feature pr.  its not much effort to make a subsystem

Reply:
especially given these arent needed prior to round start they can be late loaded, i've been trying as much as possible to shift stuff from before/during world/New() to subsystem init to reduce loading lock ups.

Reply:
the fortune cookies pick from the list, the previous implementation was broken, this is a fix PR too.

Reply:
This is a fix PR, not a features one. Nor a fix for his own feature.
Requesting for a subsystem is way out of scope.